server/db/models/incidents.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  CREATE_TABLE: `
+  CREATE TABLE IF NOT EXISTS incidents (
```

Comment:
can you separate these queries to the actual codes so that they can be imported here from different files?